### PR TITLE
Fix a few things in the development environment.

### PR DESCRIPTION
**Pull request by sanason**

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -8,7 +8,7 @@
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
-Dotenv::Railtie.load if Rails.env.development? || Rails.env.test?
+Dotenv::Rails.load if Rails.env.development? || Rails.env.test?
```
Comment (sanason) [inline config/application.rb:11]: Eliminates this warning on startup:

> DEPRECATION WARNING: Dotenv::Railtie is deprecated! Use Dotenv::Rails instead.

File: docker-compose.yml [hunk 2]
```diff
@@ -27,7 +27,9 @@ services:
     volumes:
       - pg_data:/var/lib/postgresql/data
   redis:
-    image: redis
+    image: redis:6.2
```
Comment (sanason) [inline docker-compose.yml:30]: Matches the Redis version we're on in Cloud.gov.

File: .env.test [hunk 3]
```diff
@@ -1,3 +1,6 @@
+# Postgres database
+DATABASE_URL=postgres://postgres:changeme@localhost:5432
```
Comment (ryanwoldatwork) [inline .env.test:2]: I think this line is impacting the test run.

**Review by ryanwoldatwork — APPROVED** [review]
looks once once tests pass

**Pull request merged**